src/components/Values.js

```diff
@@ -1,8 +1,13 @@
 import React from "react";
-import { StyledValues } from "../styles/StyledValues";
+import { StyledValues, StyledBonus } from "../styles/StyledValues";
 
-function Values({ points }) {
-  return <StyledValues>{points}</StyledValues>;
+function Values({ points, bonus, bonusColor }) {
+  return (
+    <StyledValues>
+      {points}
+      <StyledBonus color={bonusColor}>{bonus}</StyledBonus>
```

Comment:
I don't think we should render this component if a card doesn't have a bonus value. One way to do this would be to change the card.bonus value to false instead of an empty string for cards without a bonus. Then you could add a conditional here. Something like:  

{bonus ? <StyledBonus color={bonusColor}>{bonus}</StyledBonus> : {""}}

Reply:
yep. much better

Reply:
An empty string is falsy, so that would work too, but I think explicitly setting false is a cleaner way to do this